docs/reference/esql/functions/layout/signum.asciidoc

```diff
@@ -0,0 +1,15 @@
+// This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
```

Comment:
If you link to this file in `math-functions.asciidoc` then we'll render the generated docs.

Reply:
Would you mind updating the `package-info.java` file with instructions to do this? I should have done it when I wrote the code generation, but I forgot.

Reply:
Done and done! the `package-info.java` instructions are great btw!